### PR TITLE
feat: Product detail layout

### PR DESCRIPTION
**Pull request by mikemurray**
Issue: #18 
Impact: **minor**
Type: **feat**

## Issue

Add basic layout for the product detail page

## Solution

Using the Material UI grid, create a simple, 2 column flex layout, centered with a max-width of 1920px.

**This is just the basic layout so there's a place to put the components to test when working on the other tickets related to the PDP.**

## Breaking

none

## Testing

1. Start the app and navigate to the product detail page: `http://localhost:4000/product/1234`
1. Be amazed

### CODE REVIEW

File: src/pages/product.js [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
 import withShop from "containers/shop/withShop";
 import Layout from "components/Layout";
+import ProductDetailComponent from "components/ProductDetail";
 
 @withData
 @withRoot
 @withShop
 @inject("uiStore")
 @observer
 class ProductDetail extends Component {
```
Comment (nnnnat) [inline src/pages/product.js]: Should we rename this component to be `ProductDetailPage` and rename the `ProductDetail` import

File: src/components/ProductDetail/ProductDetail.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  }
+});
+
+/**
+ * Product detail component
+ * @name ProductDetail
+ * @param {Object} props Component props
+ * @returns {React.Component} React component node that represents a product detail view
+ */
+const ProductDetail = ({ classes, theme }) => (
```
Comment (nnnnat) [inline src/components/ProductDetail/ProductDetail.js]: Should this be a class instead of a function?

**Comment by mikemurray** [comment]
@nnnnat made the suggested changes. Ready for a re-review.

**Comment by mikemurray** [comment]
Also updated the snapshot, has changed it from a functional to a class component changed some of the auto-generated class names.

**Pull request merged**